python/smqtk/representation/descriptor_element/postgres.py

```diff
@@ -154,6 +155,20 @@ def __init__(self, type_str, uuid,
         self.db_user = db_user
         self.db_pass = db_pass
 
+        self.psql_helper = PsqlConnectionHelper(db_name, db_host, db_port,
+                                                db_user, db_pass,
+                                                self.multiquery_batch_size,
```

Comment:
This attribute does not exist in this class.  Copy/paste error from PostgresDescriptorIndex?

Reply:
Adding this to the constructor will necessitate an update of the ``get_config`` method.

Reply:
Wow, I'm sorry about my last review here... We don't actually need to add ``multiquery_batch_size`` to the psql descriptor element becuase it is never going to be doing batch operations. We should be able to safely pass ``1`` here.